src/main/java/betterquesting/api/questing/tasks/IItemTask.java

```diff
@@ -10,4 +11,6 @@ public interface IItemTask extends ITask
 {
 	boolean canAcceptItem(UUID owner, DBEntry<IQuest> quest, ItemStack stack);
 	ItemStack submitItem(UUID owner, DBEntry<IQuest> quest, ItemStack stack);
+
+    default void retrievalItems(ParticipantInfo pInfo, DBEntry<IQuest> quest, ItemStack[] items) {}
```

Comment:
`retrieval` is noun, `retrieve` is better imao

Reply:
done